src/databricks/labs/pytester/fixtures/connect.py

```diff
@@ -11,22 +11,30 @@ def spark(ws: WorkspaceClient):
     """
     Get Databricks Connect Spark session. Requires `databricks-connect` package to be installed.
 
+    To enable serverless set the local environment variable DATABRICKS_SERVERLESS_COMPUTE_ID to auto.
```

Comment:
Please link documentation

```suggestion
    To enable serverless set the local environment variable `DATABRICKS_SERVERLESS_COMPUTE_ID` to `"auto"`.
```